pkg/util/request_validation.go

```diff
@@ -0,0 +1,33 @@
+package util
```

Comment:
maybe this could all live in `webhook` as well but I thought it weird for gator to start taking on dependencies into it; t

factoring the functionality out here and having `webhookd` and `gator` call into it felt a little bit cleaner

Reply:
`util` takes one more for the team!

IMO this is fine, though at some point (forever to be pushed out), we should evaluate whether `util` can be better classified.